server/src/main/java/org/opensearch/tracer/TracerFactory.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.tracer;
+
+import io.opentelemetry.api.OpenTelemetry;
+import org.opensearch.threadpool.ThreadPool;
+
+public class TracerFactory {
```

Comment:
We need to define the disabled state of Tracer.

Reply:
done